coldfront/core/statistics/templates/job_detail.html

```diff
@@ -0,0 +1,167 @@
+                    <td>
+                        <a href="{% url 'project-detail' job.accountid.pk %}">
+                            <span class="accessibility-link-text">Project {{ job.accountid.name }}</span>
+                            {{ job.accountid.name }}
+                        </a>
+                    </td>
+                </tr>
+
+                <tr>
+                    <th scope="row" class="text-nowrap">Job Status:</th>
+                    <td>
+                        {% with status=job.jobstatus %}
+                            {% if status in status_danger_list %}
+                                <span class="badge badge-danger">{{ status }}</span>
+                            {% elif "CANCEL" in status %}
+                                <span class="badge badge-danger">CANCELLED</span>
+                            {% elif status == "COMPLETING" %}
+                                <span class="badge badge-success">COMPLETED</span>
+                            {% elif status == "RUNNING" %}
+                                <span class="badge badge-success">{{ status }}</span>
+                            {% elif status in status_warning_list %}
+                                <span class="badge badge-warning">{{ status }}</span>
+                            {% else %}
+                                <span class="badge badge-info">{{ status }}</span>
+                            {% endif %}
+                        {% endwith %}
+                    </td>
+                </tr>
+                <tr>
+                    <th scope="row" class="text-nowrap">Submit Date:</th>
+                    <td>
+                        {{ job.submitdate }}
+                    </td>
+                </tr>
+                <tr>
+                    <th scope="row" class="text-nowrap">Start Date:</th>
+                    <td>
+                        {{ job.startdate }}
+                    </td>
+                </tr>
+                <tr>
+                    <th scope="row" class="text-nowrap">End Date:</th>
+                    <td>
+                        {{ job.enddate }}
+                    </td>
+                </tr>
+                <tr>
+                    <th scope="row" class="text-nowrap">Partition:</th>
+                    <td>
+                        {{ job.partition }}
+                    </td>
+                </tr>
+                <tr>
+                    <th scope="row" class="text-nowrap">Nodes:</th>
+                    <td>
+                        {{ nodes}}
+                    </td>
+                </tr>
+                <tr>
+                    <th scope="row" class="text-nowrap">
+                      Service Units
+                      <a href="#" data-toggle="popover" title="Service Units" data-trigger="hover" data-content=
+                              "For RUNNING jobs, the amount reflects the estimated number of service units
```

Comment:
RUNNING and COMPLETED both have the same badge rn, do we want them to have different badges. I think it would probably help users differentiate between the two.

Reply:
"primary" or "info" seem reasonable to me.

https://getbootstrap.com/docs/4.0/components/badge/#contextual-variations